rules/TypeDeclaration/Rector/ClassMethod/ReturnTypeFromStrictScalarReturnExprRector.php

```diff
@@ -36,35 +45,91 @@ public function __construct(
     public function getRuleDefinition(): RuleDefinition
     {
         return new RuleDefinition('Change return type based on strict scalar returns - string, int, float or bool', [
-            new CodeSample(
+            new ConfiguredCodeSample(
+                <<<'CODE_SAMPLE'
+final class SomeClass
+{
+    public function foo($value)
+    {
+        if ($value) {
+            return 'yes';
+        }
+
+        return 'no';
+    }
+
+    public function bar(string $value)
+    {
+        return strlen($value);
+    }
+}
+CODE_SAMPLE
+                ,
                 <<<'CODE_SAMPLE'
 final class SomeClass
 {
-    public function run($value)
+    public function foo($value): string
     {
         if ($value) {
             return 'yes';
         }
 
         return 'no';
     }
+
+    public function bar(string $value): int
+    {
+        return strlen($value);
+    }
 }
 CODE_SAMPLE
+                ,
+                [
+                    self::HARD_CODED_ONLY => true,
+                ]
+            ),
+            new ConfiguredCodeSample(
+                <<<'CODE_SAMPLE'
+final class SomeClass
+{
+    public function foo($value)
+    {
+        if ($value) {
+            return 'yes';
+        }
+
+        return 'no';
+    }
 
+    public function bar(string $value)
+    {
+        return strlen($value);
+    }
+}
+CODE_SAMPLE
                 ,
                 <<<'CODE_SAMPLE'
 final class SomeClass
 {
-    public function run($value): string
+    public function foo($value): string
     {
         if ($value) {
             return 'yes';
         }
 
         return 'no';
     }
+
+    public function bar(string $value)
+    {
+        return strlen($value);
+    }
 }
 CODE_SAMPLE
+                ,
+                [
+                    self::HARD_CODED_ONLY => true,
```

Comment:
use classname::constant instead for easy copy paste: 

```
ReturnTypeFromStrictScalarReturnExprRector::HARD_CODED_ONLY => false

Reply:
Done 👍🏻